openapi/openapi.yaml

```diff
@@ -0,0 +1,470 @@
+      responses:
+        201:
+          description: Facilitation created
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Facilitation'
+        400:
+          $ref: '#/components/responses/InvalidRequest'
+        403:
+          $ref: '#/components/responses/Forbidden'
+        404:
+          $ref: '#/components/responses/NotFound'
+
+  /operators/{operatorId}/facilitations/{facilitationId}:
+    put:
+      tags:
+        - facilitation
+      summary: Update a facilitation
+      operationId: updateFacilitation
+      parameters:
+        - $ref: '#/components/parameters/OperatorId'
+        - $ref: '#/components/parameters/FacilitationId'
+      requestBody:
+        description: Facilitation to update
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/UpdateFacilitation'
+      responses:
+        200:
+          description: Facilitation updated
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Facilitation'
+        400:
+          $ref: '#/components/responses/InvalidRequest'
+        403:
+          $ref: '#/components/responses/Forbidden'
+        404:
+          $ref: '#/components/responses/NotFound'
+
+    delete:
+      tags:
+        - facilitation
+      summary: Delete a facilitation
+      operationId: deleteFacilitation
+      parameters:
+        - $ref: '#/components/parameters/OperatorId'
+        - $ref: '#/components/parameters/FacilitationId'
+      responses:
+        204:
+          description: Facilitation deleted
+        403:
+          $ref: '#/components/responses/Forbidden'
+        404:
+          $ref: '#/components/responses/NotFound'
+
+components:
+  parameters:
+    OperatorId:
+      name: operatorId
+      in: path
+      description: Operator id
+      required: true
+      schema:
+        type: string
+    FacilitationId:
+      name: facilitationId
+      in: path
+      description: Facilitation id
+      required: true
+      schema:
+        type: string
+
+  schemas:
+    CreateOperator:
+      type: object
+      required:
+        - fullName
+        - name
+        - taxCodeOrVat
+        - pecAddress
+        - referent
+        - description
+        - productCategory
+        - salesChannel
+      properties:
+        fullName:
+          type: string
+        name:
+          type: string
+        taxCodeOrVat:
+          type: string
+        pecAddress:
+          type: string
+        referent:
+          $ref: '#/components/schemas/Referent'
+        description:
+          type: string
+        productCategory:
+          type: array
+          items:
+            type: string
+          minItems: 1
+        salesChannel:
+          $ref: '#/components/schemas/SalesChannel'
+        websiteUrl:
+          type: string
+
+    UpdateOperator:
```

Comment:
Can we define a `CommonOperator` with all common properties? In this way every specification could be mapped as an extension of `CommonOperator` as well with `allOf` usage.